core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendConf.scala

```diff
@@ -60,6 +60,8 @@ trait ExternalBackendConf extends SharedBackendConf {
   def externalH2ODriverPort = sparkConf.getOption(PROP_EXTERNAL_DRIVER_PORT._1)
   def externalH2ODriverPortRange = sparkConf.getOption(PROP_EXTERNAL_DRIVER_PORT_RANGE._1)
   def externalExtraMemoryPercent = sparkConf.getInt(PROP_EXTERNAL_EXTRA_MEMORY_PERCENT._1, PROP_EXTERNAL_EXTRA_MEMORY_PERCENT._2)
+  def externalCommunicationBlockSizeAsBytes: Long = sparkConf.getSizeAsBytes(PROP_EXTERNAL_COMMUNICATION_BLOCK_SIZE._1, PROP_EXTERNAL_COMMUNICATION_BLOCK_SIZE._2)
```

Comment:
why do we need 2 properties with the same content?

Reply:
`externalCommunicationBlockSizeAsBytes` is utilized in code to get the value. IMHO, it's easier for user to work with a string representation e.g `10m` `10k` plus I would like to be aligned with [similar Spark settings](http://spark.apache.org/docs/latest/configuration.html)

In fact,  I don't need `externalCommunicationBlockSize`. I just added this for debugging purposes to be able to quickly figure out the original string value if it's set via a config file or environment variable.

Should I leave it or remove it? WDYT?